demos/python_demos/multi_camera_multi_person_tracking/mc_tracker/sct.py

```diff
@@ -317,15 +317,15 @@ def _compute_detections_assignment_cost(self, active_tracks_idx, detections, fea
                     reid_sim_avg = 1 - cosine(track_avg_feat, features[j])
                     reid_sim_curr = 1 - cosine(self.tracks[idx]['features'][-1], features[j])
                     reid_sim_clust = 1 - clusters_vec_distance(self.tracks[idx]['f_cluster'], features[j])
-                    reid_sim = min(reid_sim_avg, reid_sim_curr, reid_sim_clust)
+                    reid_sim = max(reid_sim_avg, reid_sim_curr, reid_sim_clust)
                 else:
                     reid_sim = 0.5
                 affinity_matrix[j, i] = iou * reid_sim
         return 1 - affinity_matrix
 
     @staticmethod
     def _area(box):
-        return max((box[2] - box[0]) * (box[3] - box[1]), 0)
+        return max(box[2] - box[0], 0) * max(box[3] - box[1], 0)
```

Comment:
Shouldn't it be `abs((box[2] - box[0]) * (box[3] - box[1]))`?

Reply:
No. This method should return 0 in case of invalid box. Previously it returned 0 only if height or width was <= 0, but case then both h and w are negative wasn't cached since h*w > 0. Now the method considers h and w separately.